content/22.vaccines-trad.md

```diff
@@ -424,31 +425,104 @@ These data are retrieved from Our World in Data [@url:https://ourworldindata.org
+As a result, once they began producing Nuvaxovid, they struggled to establish a stable relationship with contractors who could produce the vaccine [@doi:10.1136/bmj.n2965], especially given the challenge of producing vaccines at scale [@doi:10.1126/science.abc9588].
+Additionally, Novavax was not able to meet the purity standards laid out by the FDA, according to an investigation by _Politico_ in late 2021 [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
+Eventually, the manufacturing issues were resolved [@url:https://www.cnbc.com/2022/05/13/novavax-confident-covid-vaccine-will-receive-fda-authorization-in-june-after-delays.html], and Nuvaxovid has since been authorized by the WHO [@url:https://www.who.int/news/item/17-12-2021-who-lists-9th-covid-19-vaccine-for-emergency-use-with-aim-to-increase-access-to-vaccination-in-lower-income-countries] and by political entities, including the United Kingdom [@url:https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra], the E.U. [@ema-nuvaxovid], and the U.S. [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-emergency-use-novavax-covid-19-vaccine-adjuvanted].
+These delays obstructed some of the goals of the vaccine development program, which was undertaken with significant investment from the U.S. government through Operation Warp Speed [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
+ Novavax was supposed to provide over a billion doses of Nuvaxovid to countries around the world through the COVID-19 Vaccines Global Access (COVAX) Facility [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
+However, following the delays, Gavi (which oversees COVAX) terminated the agreement, leading to ongoing legal disagreements between the nonprofit and Novavax las of late 2022 [@url:https://www.reuters.com/business/healthcare-pharmaceuticals/gavi-says-it-is-not-breach-novavax-vaccine-deal-2022-11-22; @url:https://www.cnbc.com/2022/05/13/novavax-confident-covid-vaccine-will-receive-fda-authorization-in-june-after-delays.html].
+
+As with other vaccines, the question of how well Nuvaxovid continues to provide protection as SARS-CoV-2 evolves has been raised.
+_Post hoc_ analysis in the phase III trial indicated a VE of 86.3% against the Alpha variant (identified based on the presence of the 69–70del polymorphism) and 96.4% against viral specimens lacking the 69-70del polymorphism [@doi:10.1056/NEJMoa2107659].
+In the second phase III trial [@doi:10.1101/2021.10.05.21264567], whole-genome sequencing was obtained from 61 of the 77 observed cases, and 79% of infections were identified as a VOC or variant of interest (VOI) known at the time of the study.
```

Comment:
Is VOI necessary here?

Reply:
They always say VOC/VOI in the study so I was thinking it might be safer to follow suit!